VMEncryption/main/oscrypto/rhel_72/encryptstates/PatchBootSystemState.py

```diff
@@ -75,9 +75,29 @@ def enter(self):
             self.command_executor.ExecuteInBash('for i in dev proc sys boot; do mount --move /oldroot/$i /$i; done')
 
             extension_full_name = 'Microsoft.Azure.Security.' + CommonVariables.extension_name
+            extension_versioned_name = 'Microsoft.Azure.Security.' + CommonVariables.extension_name + '-' + CommonVariables.extension_version
+            test_extension_full_name = CommonVariables.test_extension_publisher + CommonVariables.test_extension_name
+            test_extension_versioned_name = CommonVariables.test_extension_publisher + CommonVariables.test_extension_name + '-' + CommonVariables.extension_version
```

Comment:
are these test hooks safe to include in production

Reply:
Yes Eric. My intention was to save the log files from the pivot root env in the test extension run. As of now these log files are not retained and can sometime make debugging difficult in case of failures. In production extension we will see a failure in logs saying the files cannot be found but it will not fail the encryption operation. I will enable suppress logging option for test extension log copy command so that this operation is not visible in production extension log.